eng/performance/benchmark_jobs.yml

```diff
@@ -65,14 +65,15 @@ jobs:
         strategy:
           matrix:
             ${{ each framework in parameters.frameworks }}:
-              coreclr_${{ framework }}:
-                _Category: coreclr
+              full_opt_${{ framework }}:
```

Comment:
I worry about merging coreclr and corefx tests all into one leg. The perf legs that run the full suite with coreclr and corefx separate already run for up to 2 hours (coreclr) and up to 3 hrs (corefx). Merging them together is going to make us go up to 5 hours. At the very least, you'll want to up the workitemtimeout below. But really, I think we should keep them as distributed as possible.

Reply:
I agree with @adiaaida here. Do not run all tests in the same leg.

Reply:
Can we do a timing comparison between the merged set and just corefx? Are these the 400 longest running benchmarks or the 400 shortest running benchmarks?

Reply:
@adiaaida I am going to run all of them tonight and report the results tomorrow